docs/docs/how-tos/nebari-local.md

```diff
@@ -1,4 +1,202 @@
+Since it is a regular text file, you can edit it manually if you are unhappy with the choices you made during initialization, or delete it and start over again by re-running `nebari init`.
+
+:::tip
+You can also use the new Guided Init wizard for a step-by-step initialization process.
+
+```bash
+nebari init --guided-init
+```
+
+:::
+
+<!-- TODO: Add link to advances settings doc -->
+<!-- For additional information about the `nebari-config.yaml` file and extra flags that allow you to configure the initialization process, see the
+[Understanding the nebari-config.yaml file]() documentation. -->
+
+### Setting up a DNS record for the domain
+
+In order to deploy the infrastructure, we need to set up a DNS record for the `<domain>` you provided during initialization. This will be required for the infrastructure to be able to resolve the domain name. The local deployment sets the Ingress IP to `172.18.1.100` and assigns this to your domain.
+
+To make this easier, open your `/etc/hosts` file with your favorite text editor and add the following line:
+
+```bash
+172.18.1.100 <domain>
+```
+
+:::tip
+If you are a Linux user, you can use the `sudo` command to gain root privileges and then run the following command to add the above line to your `/etc/hosts` file:
+
+```bash
+sudo echo "172.18.1.100  <domain>" | sudo tee -a /etc/hosts
+```
+:::
+
+## Deploying Nebari
+
+With the `nebari-config.yaml` configuration file now created, Nebari can be deployed for the first time with:
+
+```bash
+nebari deploy -c nebari-config.yaml --disable-prompt
+```
+
+If the deployment is successful, you will see the following output:
+
+```bash
+[terraform]: Nebari deployed successfully
+Services:
+ - argo-workflows -> https://projectname.domain/argo/
+ - conda_store -> https://projectname.domain/conda-store/
+ - dask_gateway -> https://projectname.domain/gateway/
+ - jupyterhub -> https://projectname.domain/
+ - keycloak -> https://projectname.domain/auth/
+ - monitoring -> https://projectname.domain/monitoring/
+Kubernetes kubeconfig located at file:///tmp/NEBARI_KUBECONFIG
+Kubecloak master realm username=root *****
+...
+```
+
+#### Verify the local deployment
+
+Finally, if everything is set properly you should be able to cURL the JupyterHub Server. Run
+
+```
+curl -k https://projectname.domain/hub/login
+```
+
+It's also possible to visit `https://projectname.domain` in your web browser to select the deployment.
+As default for a local deployment the https certificates generated during deployments aren't signed by a recognized [Certificate Authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority) and are self-signed by [Traefik](https://github.com/traefik/traefik) instead.
+
+To switch the default behavior and use a [Let's Encrypt](https://letsencrypt.org/) signed certificate instead, you can update the following section in your `nebari-config.yaml` file, and then re-run `nebari deploy` as shown above:
```

Comment:
If I recall correctly, the SSL certs don't work for local deployments. 

Can you confirm this @viniciusdc?

Reply:
Uhm, I would need to re-validate that info with a local install to make sure as well... but if I record correctly from previous deployments, I know that both Traefik/Let's Encrypts associated certs works just fine. Assuming in the case of Traefik that works means having to type `thisisunsafe`

Reply:
I've opened #243 to track this discussion to unblock this PR. :)